### DISCUSSION TITLE
BNT 4

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#480] [Show and tell]
### Channel ID

BNT4.bg

### Stream URL (optional)

https://de1.glebul.com/hls/bnt-4/index.m3u8?e=1692278061&hash=Wn55mSwnr9DUpxV_Acf6vg

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://de1.glebul.com/hls/bnt-4/index.m3u8?e=1692278061&hash=Wn55mSwnr9DUpxV_Acf6vg (offline)
